tests/helpers/boring_model.py

```diff
@@ -87,22 +87,26 @@ def training_step(...):
         """
         super().__init__()
         self.layer = torch.nn.Linear(32, 2)
+        # self.train_mse = MeanSquaredError()
+        # self.val_mse = MeanSquaredError()
 
     def forward(self, x):
         return self.layer(x)
 
-    def loss(self, batch, prediction):
+    def loss(self, batch, preds):
         # An arbitrary loss to have a loss that updates the model weights during `Trainer.fit` calls
-        return torch.nn.functional.mse_loss(prediction, torch.ones_like(prediction))
+        return torch.nn.functional.mse_loss(preds, torch.ones_like(preds))
 
     def step(self, x):
         x = self(x)
         out = torch.nn.functional.mse_loss(x, torch.ones_like(x))
         return out
 
     def training_step(self, batch, batch_idx):
-        output = self(batch)
-        loss = self.loss(batch, output)
+        pred = self(batch)
+        loss = self.loss(batch, pred)
```

Comment:
Can you revert these changes as the renaming shouldn't be necessary and only increases diff?